plugins/inventory/now.py

```diff
@@ -442,8 +454,23 @@ def set_hostvars(self, host, record, columns):
                 "Invalid column names: {0}.".format(", ".join(missing))
             )
 
+        show_values = self.get_option("show_values")
```

Comment:
please pass show_values to `set_hostvars` instead of looking into the module. 

I will advise passing a default value like `def set_hostvars(self, host, record, columns, show_values=False)`